[WPE] Minor fixes and spelling in cross-toolchain helper

philn opened the pull request:
#### 9c589630735a0f5d75d3b930ab4a6bbdfb632145
<pre>
[WPE] Minor fixes and spelling in cross-toolchain helper
<a href="https://bugs.webkit.org/show_bug.cgi?id=251520">https://bugs.webkit.org/show_bug.cgi?id=251520</a>

Reviewed by Carlos Alberto Lopez Perez.

* Tools/Scripts/cross-toolchain-helper: Fix a few spelling issues and allow the script to be used
from third-party Python scripts.

Canonical link: <a href="https://commits.webkit.org/259758@main">https://commits.webkit.org/259758@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/eeceed994d7befdc6cfe1ff16273297a7a76250d

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/105838 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/14906 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/38686 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/115036 "Built successfully") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/175160 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/109740 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/16325 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/6105 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/98071 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/114811 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/111587 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/12418 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/95392 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/39969 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/94284 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/27034 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/81621 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/8177 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/28386 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/8659 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/4973 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/14286 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/47934 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/6748 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/10216 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

clopez commented on Tools/Scripts/cross-toolchain-helper:
```diff
@@ -546,7 +546,7 @@ def maybe_quote_run_cmd(argv_list, cmd_key):
 
 
-def main():
+def main(*args):
```
I think this should be without asterisk.
If we use the asterisk here then we are passing to  `maybe_quote_run_cmd()` a tuple that contains inside as element zero the list instead of passing directly the list.

philn committed:
[WPE] Minor fixes and spelling in cross-toolchain helper
https://bugs.webkit.org/show_bug.cgi?id=251520

Reviewed by Carlos Alberto Lopez Perez.

* Tools/Scripts/cross-toolchain-helper: Fix a few spelling issues and allow the script to be used
from third-party Python scripts.

Canonical link: https://commits.webkit.org/259758@main
